db/migrate/20250325145055_add_display_in_cv_option.rb

```diff
@@ -0,0 +1,9 @@
+class AddDisplayInCvOption < ActiveRecord::Migration[8.0]
+  def change
+    tables = [:activities, :advanced_trainings, :educations, :projects]
+    tables.each do |table|
+      add_column table, :display_in_cv, :boolean, default: true
+    end
+    add_column :people, :display_competence_notes_in_cv, :boolean, default: true
```

Comment:
Please add the not null validation on the database level here as well and then we are fine :)